e2e/aks_model.go

```diff
@@ -243,7 +243,7 @@ func createPrivateAzureContainerRegistry(ctx context.Context, t *testing.T, reso
 	}
 
 	t.Logf("Private Azure Container Registry created\n")
-	if err := addCacheRuelsToPrivateAzureContainerRegistry(ctx, t, config.ResourceGroupName, config.PrivateACRName); err != nil {
+	if err := addCacheRuelsToPrivateAzureContainerRegistry(ctx, t, config.ResourceGroup, config.PrivateACRName); err != nil {
```

Comment:
I would keep Name, sometimes api requires ResourceGroupID...  Name and ID are the two strings that are often expected and not being clear about it could results in user errore